content/en/releases/platforms.md

```diff
@@ -0,0 +1,336 @@
+| `386-linux`     |                    | :heavy_check_mark: |        |
+| `ppc64le-linux` |                    | :heavy_check_mark: |        |
+| `s390x-linux`   |                    | :heavy_check_mark: |        |
+| `386-windows`   |                    | :heavy_check_mark: |        |
+| `amd64-windows` |                    | :heavy_check_mark: |        |
+
+### Removed platforms
+
+The following platforms have been removed from building officially in
+Kubernetes:
+
+- **Platform:** `arm-linux`
+  - **Reason:** Due to relocation problems on linking breaking the
+    `build-master` job.
+  - **Resources:**
+    - https://github.com/kubernetes/kubernetes/pull/115742
+    - https://github.com/kubernetes/kubernetes/issues/116492
+    - https://github.com/kubernetes/kubernetes/issues/115738
+    - https://github.com/kubernetes/kubernetes/issues/115675
+
+## Platforms Guide
+
+This section outlines the necessary steps to either add or remove supported
+platform builds in Kubernetes.
+
+### Adding supported platforms
+
+The default Kubernetes platform is `linux/amd64`. This platform is fully tested,
+where build and release systems initially supported only that. A while ago we
+started an [effort to support multiple architectures][0]. As part of this
+effort, we added support in our build and release pipelines for the
+architectures `arm`, `arm64`, `ppc64le` and `s390x` on different operating
+systems like Linux, Windows and macOS.
+
+[0]: https://github.com/kubernetes/kubernetes/issues/38067
+
+The main focus was to have binaries and container images to be available for
+these architectures/operating systems. Contributors should be able to to take
+these artifacts and set up CI jobs to adequately test these platforms.
+Specifically to call out the ability to run conformance tests on these
+platforms.
+
+Target of this document is to provide a starting point for adding new platforms
+to Kubernetes from a SIG Architecture and SIG Release perspective. This does not
+include release mechanics or supportability in terms of functionality.
+
+#### Step 1: Building
+
+The container image based build infrastructure should support this architecture.
+This implicitly requires the following:
+
+- golang should support the platform
+- All dependencies, whether vendored or run separately, should support this
+  platform
+
+In other words, anyone in the community should be able to use our build infra to
+generate all artifacts required to stand up Kubernetes.
+
+More information about how to build Kubernetes can be found in [the build
+documentation][1].
+
+[1]: https://github.com/kubernetes/kubernetes/tree/3f7c09e/build#building-kubernetes
+
+#### Step 2: Testing
+
+It is not enough for builds to work as it gets bit-rotted quickly when we vendor
+in new changes, update versions of things we use etc. So we need a good set of
+tests that exercise a wide battery of jobs in this new architecture.
+
+A good starting point from a testing perspective are:
+
+- unit tests
```

Comment:
Do we actually get something from running unit tests on a given architecture? I think it's mainly E2E tests and node E2E tests that we need for non-amd64 architectures

Reply:
Can we add the following as a list?
- The arch must have official go builds
- The arch must be supported by Debian and included in LTS.
- A list of go/apps that must be built for the arch.

Reply:
> Do we actually get something from running unit tests on a given architecture? I think it's mainly E2E tests and node E2E tests that we need for non-amd64 architectures

it gives us higher confidence that we don't have platform specific integer sizes and issues like that, and that all the code compiles. it's a low baseline, not sufficient on it's own, but if we can't even get unit test signal then ...